aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -1778,4 +1778,12 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
     override fun dispatchHoverEvent(event: MotionEvent): Boolean {
         return if (accessibilityDelegate.onHoverEvent(event)) true else super.dispatchHoverEvent(event)
     }
+
+    fun setUrlClickable(urlClickable: Boolean) {
+        EnhancedMovementMethod.clickableUrlSpan = urlClickable
+    }
+
+    fun setOnUrlClickListener(listener: EnhancedMovementMethod.OnUrlClickListener?) {
```

Comment:
Would you mind adding a configuration method to the `Aztec` class, similar to [this](https://github.com/adrielcafe/AztecEditor-Android/blob/f5c71e91f338aaf83554f2c75e5eadb4cc9da629/aztec/src/main/kotlin/org/wordpress/aztec/Aztec.kt#L101-L105)?